identity-service/src/notifications/index.js

```diff
@@ -97,11 +102,20 @@ class NotificationProcessor {
 
     // Email notification queue
     this.emailQueue.process(async (job, done) => {
-      logger.info('about to processEmailNotifications')
+      logger.info('processEmailNotifications')
       await processEmailNotifications(expressApp, audiusLibs)
       done()
     })
 
+    // Announcement push notifications queue
+    this.announcementQueue.process(async (job, done) => {
+      await pushAnnouncementNotifications()
+      // Delay 3s
+      await new Promise(resolve => setTimeout(resolve, 3000))
```

Comment:
I think we should make this like every 10 minutes. i dont think this needs to run that frequently